src/hotspot/share/runtime/atomic.hpp

```diff
@@ -307,8 +307,8 @@ class Atomic : AllStatic {
   struct AddImpl;
 
   // Platform-specific implementation of add.  Support for sizes of 4
-  // bytes and (if different) pointer size bytes are required.  The
-  // class must be default constructable, with these requirements:
+  // and 8 bytes are required.  The class must be default constructable,
```

Comment:
Comment change seems unnecessary.

Reply:
I found that I had to add 64-bit atomic add support on 32-bit platforms (working on it), so I changed the comment to match the similar cmpxchg one.

Reply:
Reverted, since 64-bit atomic add is no longer needed.